xdebug_filter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+if (function_exists('xdebug_set_filter')) {
+    xdebug_set_filter(
+        XDEBUG_FILTER_CODE_COVERAGE,
+        XDEBUG_PATH_INCLUDE,
+        [ __DIR__ . DIRECTORY_SEPARATOR . 'src' . DIRECTORY_SEPARATOR ]
+    );
+    xdebug_set_filter(
```

Comment:
Isn't this managed by PHPUnit already based on the configuration in phpunit.xml ? Or is it not supported for the phpt runner ?

Reply:
I don't know exactly, maybe is due to PHPT; locally, as said in the commit message, with this I observed a reduction of execution time from 26 seconds to only 3.